src/pathfinding_service/api.py

```diff
@@ -225,30 +234,44 @@ def get(self, token_network_address: TokenNetworkAddress) -> Tuple[dict, int]:
 
 
 class InfoResource(PathfinderResource):
-    version = pkg_resources.get_distribution('raiden-services').version
+    version = pkg_resources.get_distribution("raiden-services").version
 
     def get(self) -> Tuple[dict, int]:
         price = 0
-        settings = 'PLACEHOLDER FOR PATHFINDER SETTINGS'
-        operator = 'PLACEHOLDER FOR PATHFINDER OPERATOR'
-        message = 'PLACEHOLDER FOR ADDITIONAL MESSAGE BY THE PFS'
+        settings = "PLACEHOLDER FOR PATHFINDER SETTINGS"
+        operator = "PLACEHOLDER FOR PATHFINDER OPERATOR"
+        message = "PLACEHOLDER FOR ADDITIONAL MESSAGE BY THE PFS"
 
         return (
             {
-                'price_info': price,
-                'network_info': {
-                    'chain_id': self.pathfinding_service.chain_id,
-                    'registry_address': self.pathfinding_service.registry_address,
+                "price_info": price,
+                "network_info": {
+                    "chain_id": self.pathfinding_service.chain_id,
+                    "registry_address": self.pathfinding_service.registry_address,
                 },
-                'settings': settings,
-                'version': self.version,
-                'operator': operator,
-                'message': message,
+                "settings": settings,
+                "version": self.version,
+                "operator": operator,
+                "message": message,
             },
             200,
         )
 
 
+class DebugEndpoint(PathfinderResource):
+    def get(  # pylint: disable=no-self-use
+        self, token_network_address: TokenNetworkAddress, source_address: Address
```

Comment:
This needs to able to optionally handle `target_address` as well. 
The SP `assert_pfs_history` task passes that as the last path segment if `to: <int>` is given in the task config.